dds_registration/models.py

```diff
@@ -88,7 +88,15 @@ def __init__(self, *args, **kwargs):
 
 
 class Membership(models.Model):
+    MEMBERSHIP_TYPES = [
```

Comment:
The previous `` definition was:

```
    MEMBERSHIP_TYPES = (
        ("NORMAL_CREDIT_CARD", "Normal membership - Pay by credit card"),
        ("NORMAL_INVOICE", "Normal membership - Pay by invoice"),
        ("ACADEMIC_CREDIT_CARD", "Academic membership - Pay by credit card"),
        ("ACADEMIC_INVOICE", "Academic membership - Pay by invoice"),
    )
```

And it was supposed mostly to determine the payment option.

How would we determine this now?

Reply:
All payment information should *only* be on the `Invoice` class. In this case, we would have two registration options - academic or normal. The question of *how the payment is made* is a different one, this should only be on the `Invoice` model.

Reply:
Should we add the `registration_type` field with choices ('Academic', 'Normal')? Where to? To the `Registration` model? Or to the `Invoice`?

Reply:
So, the way I see this is that we have a single unified process:

- User wants to register or apply for an event. The click the link from an email or from the homepage.
- They fill out the registration form. Note that we don't have this perfectly figure out yet, as we will need to ask multiple questions (see e.g. [the Autumn School form](https://docs.google.com/forms/d/e/1FAIpQLSdj6guYxEuuYgHymX2VNlvyKoQkIa8CCa_v-KAoT1k30N2f9Q/viewform)).
- The last part of that form is the kind of registration (`RegistrationOption`). There can be options for amount and currency (e.g. single or double room, in USD or CAD). When the form is complete and any form errors have been fixed, a `Registration` object is created. The status is set to either SUBMITTED or PAYMENT-PENDING, depending on the event type (we need to add a new boolean column to event for this, maybe `application_review`).
- The user is redirected to the invoice creation form. It is *here* that they can choose to pay via credit card - this is a link to another page controlled by Stripe. Otherwise, they will enter the data we don't already have (invoice-specific stuff), as we generate a PDF for them. We need to pass the registration option in the GET parameters.

Does that make sense?

Reply:
> Should we add the `registration_type` field with choices ('Academic', 'Normal')? Where to? To the `Registration` model? Or to the `Invoice`?

Sorry, this is for membership - yes, we need to add another element to `MEMBERSHIP_TYPE`, `("ACADEMIC", "Academic")`.